gulpfile.js

```diff
@@ -255,7 +304,13 @@ gulp.task('run-test-e2e', function() {
 });
 
 gulp.task('test', function() {
-  runSequence('build', 'webserver', 'test-integration', 'stop-webserver');
+  runSequence(
+    'build',
+    'tests-unit',
+    'webserver',
+    'test-integration',
+    'stop-webserver'
+  );
```

Comment:
Right, I haven't dealt with big files yet, but iirc they are usually split in several files/modules - we can try this later as well.